server/models/singleSourceOfTruthModel.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const {Schema} = mongoose;
+
+const robotMetadataSchema = new Schema({
+    robotId: String,
+    starterId: String
+});
+
+const instructionSchema = new Schema({
+    type: String,
+    name: String,
+    predecessorIds: [mongoose.Types.ObjectId],
+    successorIds: [mongoose.Types.ObjectId],
+    rpaApplication: String,
+    rpaTask: String,
+    rpaParameters: [inputParameterSchema],
+    outputVariable: Schema.Types.Mixed
+});
+
```

Comment:
I would have expected an element schema here, that is being used as an object in every instruction, marker, ...

What do you think about that?

Reply:
I wasn't too sure about it, but did what I think would fit. This schema should be edited and or built upon in the issue regarding the SSoT saving